tests/components/UserActions/UserActivity.spec.ts

```diff
@@ -5,64 +5,64 @@ import { UserActionType } from '../../../src/rest/UserProfilingEndpoint';
 import { UserActivity } from '../../../src/Index';
 import { delay, fakeUserProfileModel } from '../../utils';
 import { formatDate, formatTime, formatTimeInterval } from '../../../src/utils/time';
-
 describe('UserActivity', () => {
-    const FAKE_CLICK_EVENT = new UserAction(UserActionType.Click, new Date('1:00:00 AM'), {
+    const TEST_DATE_STRING = 'December 17, 1995 1:00:00 AM'
+    const FAKE_CLICK_EVENT = new UserAction(UserActionType.Click, new Date(TEST_DATE_STRING), {
         origin_level_1: 'relevant' + Math.random(),
         uri_hash: 'product' + Math.random(),
         c_contentidkey: '@sysurihash',
         c_contentidvalue: '' + Math.random()
     });
     FAKE_CLICK_EVENT.document = Fake.createFakeResult();
 
-    const FAKE_SEARCH_EVENT = new UserAction(UserActionType.Search, new Date('1:00:00 AM'), {
+    const FAKE_SEARCH_EVENT = new UserAction(UserActionType.Search, new Date(TEST_DATE_STRING), {
         origin_level_1: 'relevant' + Math.random(),
         cause: 'interfaceLoad'
     });
 
-    const FAKE_USER_SEARCH_EVENT = new UserAction(UserActionType.Search, new Date('1:00:00 AM'), {
+    const FAKE_USER_SEARCH_EVENT = new UserAction(UserActionType.Search, new Date(TEST_DATE_STRING), {
         origin_level_1: 'relevant' + Math.random(),
         query_expression: 'someSearch' + Math.random(),
         cause: 'searchboxSubmit'
     });
     FAKE_USER_SEARCH_EVENT.query = FAKE_USER_SEARCH_EVENT.raw.query_expression;
 
-    const FAKE_VIEW_EVENT = new UserAction(UserActionType.PageView, new Date('1:00:00 AM'), {
+    const FAKE_VIEW_EVENT = new UserAction(UserActionType.PageView, new Date(TEST_DATE_STRING), {
         origin_level_1: 'relevant' + Math.random(),
         content_id_key: '@someKey' + Math.random(),
         content_id_value: 'someValue' + Math.random()
     });
 
-    const FAKE_CUSTOM_EVENT = new UserAction(UserActionType.Custom, new Date('1:00:00 AM'), {
+    const FAKE_CUSTOM_EVENT = new UserAction(UserActionType.Custom, new Date(TEST_DATE_STRING), {
         origin_level_1: 'relevant' + Math.random(),
         event_type: 'Submit' + Math.random(),
         event_value: 'Case Submit' + Math.random()
     });
 
-    const FAKE_CUSTOM_EVENT_WITHOUT_TYPE = new UserAction(UserActionType.Custom, new Date('1:00:00 AM'), {
+    const FAKE_CUSTOM_EVENT_WITHOUT_TYPE = new UserAction(UserActionType.Custom, new Date(TEST_DATE_STRING), {
         origin_level_1: 'relevant' + Math.random(),
         event_value: 'Case Submit' + Math.random()
     });
 
     const IRRELEVANT_ACTIONS = [
```

Comment:
Do you think a 'dumber' array (i.e. with primitive) and a `map` would be better?